ros_buildfarm/templates/doc/doc_independent_docker_job.xml.em

```diff
@@ -132,10 +158,12 @@ doc_repository_name = os.path.splitext(os.path.basename(doc_repository_url))[0]
     dynamic_recipients=[],
     send_to_individuals=False,
 ))@
+@[if upload_repository_url is not None]@
```

Comment:
Just to make sure, should this be `upload_repository_branch`?

Reply:
Yes and no.  The code in https://github.com/ros-infrastructure/ros_buildfarm/pull/846/files#diff-2f3d0521af79f10d17016a49a17b0007a0cd3510e58bbc7f2de273ec30ca2e5cR187 ensures that if there is an `upload_repository_url`, there is also guaranteed to be an `upload_repository_branch`.  In order to match this snippet with the snippet above at https://github.com/ros-infrastructure/ros_buildfarm/pull/846/files#diff-f5faccbba5d43185cbbd3c046bad114c25aa1782f9a0b11c07dd6ac6939bb6e4R27, I decided to use the same conditional for both.

Essentially, it is a trade-off in readability: we can either make the two snippets match (and hence make it easier to discover that they "go together"), or we can check the variables we use in a particular snippet (making it a bit safer, but also losing the property that these go together).